classes/pdf/PDFGenerator.php

```diff
@@ -97,10 +97,14 @@ class PDFGeneratorCore extends TCPDF
      * @param bool $use_cache
      * @param string $orientation
      */
-    public function __construct($use_cache = false, $orientation = 'P')
+    public function __construct($use_cache = false, $orientation = 'P', $language = null)
```

Comment:
```suggestion
    public function __construct($use_cache = false, $orientation = 'P', Language $language = null)
```

And could you add PHPDoc, please ?